website/pages/docs/commands/operator/raft.mdx

```diff
@@ -0,0 +1,154 @@
+---
+layout: docs
+page_title: operator raft - Command
+sidebar_title: <code>raft</code>
+description: >-
+  The "operator raft" command is used to interact with the integrated Raft storage backend.
+---
+
+# operator raft
+
+This command groups subcommands for operators to manage the integrated raft storage backend.
```

Comment:
```suggestion
This command groups subcommands for operators to manage the integrated Raft storage backend.
```
We should be consistent with capitalization - it seems like Consul mostly uses 'Raft' over 'raft', so can we stick with that?

Reply:
Sure. Done.